src/nodelet/face_detection_nodelet.cpp

```diff
@@ -236,8 +236,13 @@ class FaceDetectionNodelet : public opencv_apps::Nodelet
     msg_pub_ = advertise<opencv_apps::FaceArrayStamped>(*pnh_, "faces", 1);
 
     std::string face_cascade_name, eyes_cascade_name;
+#ifdef HYDRO
```

Comment:
@k-okada Test seems failed because the default param value specified here will be overwritten by launch file: https://github.com/ros-perception/opencv_apps/blob/indigo/launch/face_detection.launch#L10-L17